SPECS/systemd/systemd-bootstrap.spec

```diff
@@ -37,6 +37,7 @@ Requires:       libgcrypt
 Requires:       lz4
 Requires:       pam
```

Comment:
Considering the errors you've sent in the e-mail, I think we need to do `Requires(post): pam` as well.

Reply:
Sure, updated